django/db/backends/base/schema.py

```diff
@@ -749,7 +751,9 @@ def _alter_field(self, model, old_field, new_field, old_type, new_type,
             self.execute(
                 self.sql_create_check % {
                     "table": self.quote_name(model._meta.db_table),
-                    "name": self.quote_name(self._create_index_name(model, [new_field.column], suffix="_check")),
+                    "name": self.quote_name(self._create_index_name(
```

Comment:
put self._create_index_name( on the next line of put the suffix="_check") parenthesis on the next line for balance